x/rollapp/keeper/state_info.go

```diff
@@ -34,6 +36,17 @@ func (k Keeper) GetStateInfo(
 	return val, true
 }
 
+func (k Keeper) MustGetStateInfo(ctx sdk.Context,
```

Comment:
why not changing `GetStateInfo` all together to panic? because of the cascading changes required all over?

Reply:
I think it make sense as u have queries for stateInfo, which u probably don't want to panic for non existing index

Reply:
what @mtsitrin said